components/support/migration/src/main/java/mozilla/components/support/migration/FennecMigrator.kt

```diff
@@ -375,7 +375,7 @@ class FennecMigrator private constructor(
     fun <T> startMigrationServiceIfNeeded(service: Class<T>) where T : AbstractMigrationService {
         if (hasMigrationsToRun()) {
             logger.debug("Has migrations to run. Starting service.")
-            context.startService(Intent(context, service))
+            context.startForegroundService(Intent(context, service))
```

Comment:
`ContextCompat.startForegroundService()`? 🤔 

I think we also need a permission for Android P:
`<uses-permission android:name=”android.permission.FOREGROUND_SERVICE”/>`

Reply:
Oh, yeah, that is right. We already have that for other reasons, but we should make that explicit in this component too.

Reply:
Updated!